PCU/src/main/java/com/coffeelovers/pcu/model/Invoice.java

```diff
@@ -11,7 +15,7 @@
 	
 	@Id
 	@GeneratedValue(strategy = GenerationType.AUTO)
-	private long id;
+	private long invliceId;
```

Comment:
@Andrew-shanan should this one be InvoiceId?

Reply:
@Andrew-shanan when you add the things you said you want to add you can fix it.